src/elements/content-sidebar/activity-feed/comment/types.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import type { AnnotationPermission, BoxCommentPermission, FeedItemStatus } from '../../../../common/types/feed';
+
+export type OnAnnotationEdit = (args: { id: string, permissions: AnnotationPermission, text?: string }) => void;
+
+export type OnCommentEdit = (args: {
+    hasMention: boolean,
+    id: string,
+    onError?: Function,
```

Comment:
Same here

Reply:
That'd be better. Let me see if I can track these down.

Reply:
Yeah, I think I looked at this initially, and unforch, it's pretty hard to nail down what was intended. `onCommentUpdate` is passed to `onCommentEdit` in `ActivityFeed`.
```
onCommentUpdate?: (
        id: string,
        text?: string,
        status?: FeedItemStatus,
        hasMention: boolean,
        permissions: BoxCommentPermission,
        onSuccess: ?Function,
        onError: ?Function,
    ) => void,
```
`ActivitySidebar's` `this.updateComment` is passed to `onCommentUpdate`, and the type there is:
```
updateComment = (
        id: string,
        text?: string,
        status?: FeedItemStatus,
        hasMention: boolean,
        permissions: BoxCommentPermission,
        onSuccess: ?Function,
        onError: ?Function,
    ): void => {
 ```
I haven't been able to find the onError and onSuccess params getting used, so I'm just really not sure if there were some more specific intentions there.